src/components/commons/Container.tsx

```diff
@@ -0,0 +1,14 @@
+import { PropsWithChildren } from "react";
+import classNames from "classnames";
+
+/**
+ * Container implementation that is part of the design language.
+ */
+export default function Container({
+  className,
+  children,
+}: PropsWithChildren<{ className?: string }>): JSX.Element {
+  return (
+    <div className={classNames("container mx-auto", className)}>{children}</div>
```

Comment:
actually we already have `clsx` let's use that? then we can remove `classNames` package
```suggestion
    <div className={clsx("container mx-auto", className)}>{children}</div>
```